core/src/main/java/org/moskito/control/core/history/service/InMemoryHistoryService.java

```diff
@@ -0,0 +1,51 @@
+package org.moskito.control.core.history.service;
+
+import org.moskito.control.config.MoskitoControlConfiguration;
+import org.moskito.control.core.history.StatusUpdateHistoryItem;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class InMemoryHistoryService implements HistoryService {
+    private final List<StatusUpdateHistoryItem> items = new LinkedList<>();
+
+    @Override
+    public synchronized void addItem(StatusUpdateHistoryItem historyItem) {
```

Comment:
isn't it better to use concurrent list here? also we usually allow lists to grow over the max size a bit, and cut them:
int maxAllowedSize =  MoskitoControlConfiguration.getConfiguration().getHistoryItemsAmount()*1.10 (+10%)
if (items.size()>maxAllowedSize())
   items = items.sublist(0, MoskitoControlConfiguration.getConfiguration().getHistoryItemsAmount())

This allows for lesser remove operations (hence lesser time in synchronized block), but the sublist may be not serializeable (probably doesn't matter here)